src/util/DbrHelper.sol

```diff
@@ -0,0 +1,335 @@
+            uint256 dbrLeft
+        )
+    {
+        if (params.sellForDola > 0) {
+            uint256 sellAmountForDola = (amount * params.sellForDola) /
+                denominator;
+
+            if (repay.percentage != 0) {
+                (dolaAmount, repaidAmount) = _sellAndRepay(
+                    sellAmountForDola,
+                    params.minOutDola,
+                    repay
+                );
+            } else {
+                dolaAmount = _sellDbr(
+                    sellAmountForDola,
+                    params.minOutDola,
+                    dolaIndex,
+                    params.toDola
+                );
+            }
+        }
+
+        if (params.sellForInv > 0) {
+            uint256 sellAmountForInv = (amount * params.sellForInv) /
+                denominator;
+            invAmount = _sellAndDeposit(
+                sellAmountForInv,
+                params.minOutInv,
+                params.toInv
+            );
+        }
+
+        // Send leftover DBR to the receiver
+        dbrLeft = dbr.balanceOf(address(this));
+        if (dbrLeft > 0) dbr.transfer(params.toDbr, dbrLeft);
+        // Send leftover DOLA to the receiver
+        uint256 dolaLeft = dola.balanceOf(address(this));
+        if (dolaLeft > 0) dola.transfer(params.toDola, dolaLeft);
+    }
+
+    /// @notice Sell DBR amount for INV and deposit into the escrow
+    /// @param amount Amount of DBR to sell
+    /// @param minOutInv Minimum amount of INV to receive
+    /// @param to Address to receive INV deposit
+    /// @return invAmount Amount of INV deposited
+    function _sellAndDeposit(
+        uint256 amount,
+        uint256 minOutInv,
+        address to
+    ) internal returns (uint256 invAmount) {
+        // Sell DBR for INV
+        _sellDbr(amount, minOutInv, invIndex, address(this));
+        // Deposit INV
+        invAmount = inv.balanceOf(address(this));
+        invMarket.deposit(to, invAmount);
+
+        emit DepositInv(msg.sender, to, invAmount);
+    }
+
+    /// @notice Sell DBR amount for DOLA and repay debt
+    /// @param amount Amount of DBR to sell
+    /// @param minOutDola Minimum amount of DOLA to receive
+    /// @param repay Repay struct with parameters for repaying debt
+    /// @return dolaAmount Amount of DOLA received after selling DBR
+    /// @return repaidAmount Actual amount of DOLA repaid
+    function _sellAndRepay(
+        uint amount,
+        uint minOutDola,
+        Repay calldata repay
+    ) internal returns (uint256 dolaAmount, uint256 repaidAmount) {
+        // Sell DBR for DOLA
+        dolaAmount = _sellDbr(amount, minOutDola, dolaIndex, address(this));
+        // Repay debt
+        repaidAmount = _repay(repay, dolaAmount);
+    }
+
+    /// @notice Repay debt
+    /// @param repay Repay struct with parameters for repaying debt
+    /// @param dolaAmount Amount of DOLA available to repay
+    /// @return repaidAmount Actual amount of DOLA repaid
```

Comment:
Just wanted to add that little bit of documentation
```suggestion
    /// @notice Repay debt
    /// @dev Must transfer any remaining DOLA out of contract after being called
    /// @param repay Repay struct with parameters for repaying debt
    /// @param dolaAmount Amount of DOLA available to repay
    /// @return repaidAmount Actual amount of DOLA repaid
```

Reply:
Fixed in [f446755](https://github.com/InverseFinance/FiRM/pull/66/commits/f44675526e646aab388f30b936371beb701648ff)